gulp/tasks/browserify-task.js

```diff
@@ -28,18 +34,27 @@ gulp.task('browserify-app', function(){
     .on('error', errorHandler)
     .pipe(source('app.js'))
     .pipe(replace(/__PACKAGE_VERSION__/g, pkgVersion))
+    .pipe(gulpif(codap, rename('app.js.ignore')))
     .pipe(gulp.dest(config.app.dest));
 });
 
 gulp.task('browserify-globals', function(){
   var b = browserify({
-    debug: !production
+    debug: !production && !noMap,
   });
+  if(nojQuery) {
+    b.exclude('jquery');
+  }
   b.transform(coffeeify);
   b.add(config.globals.src);
   return b.bundle()
     .on('error', errorHandler)
     .pipe(source('globals.js'))
+    .pipe(gulpif(nojQuery, replace(/(var)*[ ]*jQuery[ ]*=/g,
```

Comment:
You should probably change this to 
    /(var)*\s*jQuery\s*=/g
in case someone starts using tabs in the future.